db_auth_init.sql

```diff
@@ -0,0 +1,8 @@
+-- is calles while initalizing the database
+-- as has to be run as root user to set the permissions
```

Comment:
Do we need those two comments? I don't understand them.

Reply:
extended the comments